plugins/GranularPitchShifter/GranularPitchShifterEffect.cpp

```diff
@@ -96,8 +96,8 @@ bool GranularPitchShifterEffect::processAudioBuffer(sampleFrame* buf, const fpp_
 
 	for (fpp_t f = 0; f < frames; ++f)
 	{
-		const double pitch = pitchBuf ? pitchBuf->value(f) : m_granularpitchshifterControls.m_pitchModel.value();
-		const double pitchSpread = (pitchSpreadBuf ? pitchSpreadBuf->value(f) : m_granularpitchshifterControls.m_pitchSpreadModel.value()) * 0.5f;
+		const double pitch = (pitchBuf ? pitchBuf->value(f) : m_granularpitchshifterControls.m_pitchModel.value()) * (1. / 12.);
```

Comment:
Here too, but since this is double, should you?

Reply:
I forgot it's working with doubles instead of floats.